apps/web/src/lib/components/book-reader/book-reader-continuous/book-reader-continuous.svelte

```diff
@@ -111,6 +124,14 @@
 
   const destroy$ = new Subject<void>();
 
+  const sectionToElement: Map<string, HTMLElement> = new Map();
+
+  const sectionData: Map<string, SectionWithProgress> = new Map();
```

Comment:
For future reference you can actually do `const sectionData = new Map<string, SectionWithProgress>();`